alicloud/table_alicloud_ram_policy.go

```diff
@@ -0,0 +1,245 @@
+package alicloud
+
+import (
+	"context"
+	"time"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/errors"
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/ram"
+	"github.com/sethvargo/go-retry"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAlicloudRamPolicy(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:             "alicloud_ram_policy",
+		Description:      "Alibaba Cloud RAM Policy",
+		DefaultTransform: transform.FromCamel(),
+		List: &plugin.ListConfig{
+			Hydrate: listRAMPolicy,
+		},
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.AllColumns([]string{"name", "policy_type"}),
+			ShouldIgnoreError: isNotFoundError([]string{"InvalidParameter.PolicyType", "EntityNotExist.Policy", "MissingParameter"}),
+			Hydrate:           getRAMPolicy,
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "name",
```

Comment:
@bigdatasourav Should this be `name` or `policy_type`? It seems inconsistent that we shorten this field, but not the `PolicyType` field? Do we have a precedent for this on when we shorten property names? I think in the AWS plugin, when we get a field name like `certificate_name`, we usually keep it as is because often users know it as such (if the CLI, SDKs, and APIs return it as such), but I'm not sure if do something different in Alicloud usually.

Reply:
@cbruno10 I have updated it as policy_name. In alicloud for ram tables like group, role, user we kept the column as name. In AWS also for tables like backup_plan - BackupPlanName, aws_availability_zone - ZoneName, we kept the column as name. I agree that we should keep it as we are getting from the API, will follow the same.